### DISCUSSION TITLE
Restore Backup

### DISCUSSION THREAD

**Discussion by Novido** [GameServerManagers/LinuxGSM#3258] [Ideas]
It would be nice to have a way to easily restore a backup.

For example: ./gameserver restore [backupname]

**Comment by patmann03** [comment]
I would agree, I was looking for this to see how I would go about doing it... Seems I'd need to take the backup and restore over the home directory.

**Comment by dgibbs64** [comment]
Backup is in need of some TLC and is in line for a refactor in the future. But that may be a little while. You can also look at using other methods to backup https://docs.linuxgsm.com/commands/backup#alternative-backup-methods
